backend/common/src/main/java/ai/verta/modeldb/common/config/RdbConfig.java

```diff
@@ -94,6 +94,10 @@ public static String buildDatabaseServerConnectionString(RdbConfig rdb) {
       return rdb.DBConnectionURL;
     }
 
+    if (rdb.isH2()) {
+      return rdb.RdbUrl + ";DB_CLOSE_DELAY=-1";
```

Comment:
any idea why we didn't need this in UAC?

Reply:
> any idea why we didn't need this in UAC?

I have tried to run uac without this and I am getting the error. To reproduce the issue. You have to start uac using the H2 database configuration.

Reply:
but... we run UAC without this on every PR, and we don't get an error?

Reply:
> but... we run UAC without this on every PR, and we don't get an error?

It surprised me too. If we are running uac service not test cases then I get the error..

Reply:
I just started it and it worked fine. 🤔

Reply:
I do see that this helps things in the registry though. 🤔